src/main/java/org/candlepin/subscriptions/resource/api/v1/SubscriptionTableController.java

```diff
@@ -195,7 +194,7 @@ public SkuCapacityReport capacityReportBySku( // NOSONAR
       }
     }
 
-    boolean isOnDemand = SubscriptionDefinition.isPrometheusEnabled(productId.toString());
+    boolean isOnDemand = productId.isPrometheusEnabled() || productId.isPayg();
```

Comment:
Moving this logic which is repeated in several places into the ProductId would avoid the duplication:
```java
public boolean isOnDemand() {
   return isPrometheusEnabled() || isPayg();
}
```
This way if in the future, the condition changes, we would only require to change in one place.